packages/svelte/src/compiler/phases/1-parse/read/style.js

```diff
@@ -9,7 +9,7 @@ const REGEX_PERCENTAGE = /^\d+(\.\d+)?%/;
 const REGEX_NTH_OF =
 	/^(even|odd|\+?(\d+|\d*n(\s*[+-]\s*\d+)?)|-\d*n(\s*\+\s*\d+))((?=\s*[,)])|\s+of\s+)/;
 const REGEX_WHITESPACE_OR_COLON = /[\s:]/;
-const REGEX_BRACE_OR_SEMICOLON = /[{;]/;
+const REGEX_BRACE_OR_SEMICOLON = /[{;](?=(?:[^']*'[^']*')*[^']*$)/;
```

Comment:
```suggestion
const REGEX_BRACE_OR_SEMICOLON = /[{;](?=[^"']*$)/;
```

I think the extra complexity here isn't needed — just checking that it doesn't end with a quote would be sufficient. Also don't forget the `"`  character.

Reply:
Sorry about testing, I'm starting in this world of contributing to open source now and even don't know how I would write a test for that.
As for the changeset, is there a style guide or other PR that I could use as example ?

Reply:
No worries! The test case for this should be added in [parser-modern](https://github.com/sveltejs/svelte/tree/main/packages/svelte/tests/parser-modern/samples). You can read more about contributing to Svelte [here](https://github.com/sveltejs/svelte/blob/main/CONTRIBUTING.md#pull-requests). 

Unfortunately, I can't help you add a test case as I don't have write access to this PR. You can look at similar tests inside parser-modern though to understand how it works. In short, it is a snapshot test, meaning it takes an input and expects a certain output, if the generated output is different than the expected it fails.

Reply:
@navorite added a changeset and commited your sugestion. as for tests I'm still stucked